engine/Scripts/lib_build.py

```diff
@@ -0,0 +1,406 @@
+
+    os.chdir(Path(flutter_root_path))
+    os.system("python ./flutter/tools/gn " + gn_params)
+
+    if platform == "mac" or platform == "ios":
+        f = open(Path(flutter_root_path + "/out/" + output_path + "/args.gn"), 'a')
+        f.write("icu_use_data_file=false")
+        f.close()
+        os.system("ninja " + ninja_params)
+    elif platform == "windows":
+        f = open(Path(flutter_root_path + "/out/" + output_path + "/args.gn"), 'a')
+        f.write("skia_use_angle = true\nskia_use_egl = true")
+        f.close()
+        os.system("ninja " + ninja_params1)
+        os.system("ninja " + ninja_params2)
+        os.system("ninja " + ninja_params3)
+        os.chdir(Path(flutter_root_path + "/third_party/icu/flutter/"))
+        os.system("ld -r -b binary -o icudtl.o icudtl.dat")
+    elif platform == "android":
+        os.system("ninja " + ninja_params)
+
+def build_engine():
+    global flutter_root_path
+    global work_path
+    global runtime_mode
+    global platform
+    global output_path
+    global bitcode
+    dest_folder=""
+
+    print("\nStarting build engine...")
+    if platform == "windows":
+        dest_folder = "x86_64"
+    elif platform == "mac":
+        dest_folder = "osx"
+    elif platform == "android":
+        dest_folder = "android"
+        os.chdir(work_path + "/../")
+        os.system("python " + flutter_root_path + "/flutter/sky/tools/objcopy.py --objcopy " + flutter_root_path + "/third_party/android_tools/ndk/toolchains/arm-linux-androideabi-4.9/prebuilt/darwin-x86_64/bin/arm-linux-androideabi-objcopy --input " + flutter_root_path + "/third_party/icu/flutter/icudtl.dat --output icudtl.o --arch arm")
+    elif platform == "ios":
+        dest_folder = "ios"
+    if not os.path.exists(Path(work_path + "/../../com.unity.uiwidgets/Runtime/Plugins/" + dest_folder)):
+        os.makedirs(Path(work_path + "/../../com.unity.uiwidgets/Runtime/Plugins/" + dest_folder))
+    os.chdir(Path(work_path + "/../"))
+    if runtime_mode == "release":
+        if os.path.exists(Path(work_path + "/../build_release")):
+            shutil.rmtree(Path(work_path + "/../build_release"))
+        if platform == "windows":
+            os.system("bee.exe win_release")
+            copy_file(Path(work_path + "/../build_release/"), Path(work_path + "/../../com.unity.uiwidgets/Runtime/Plugins/" + dest_folder))
+        else:
+            os.system("mono bee.exe " + platform +"_release")
+            copy_file(Path(work_path + "/../build_release/"), Path(work_path + "/../../com.unity.uiwidgets/Runtime/Plugins/" + dest_folder))
+            if platform == "android":
+                if not os.path.exists(Path(work_path + "/../artifacts/rsp/backup")):
+                    os.makedirs(Path(work_path + "/../artifacts/rsp/backup"))
+                copy_file(Path(work_path + "/../artifacts/rsp/14590475716575637239.rsp"), Path(work_path + "/../artifacts/rsp/backup/14590475716575637239.rsp"))
```

Comment:
why do we use a fixed file name, i.e., 14590475716575637239.rsp ? 
we need get the file name from tundra.dag.json